travis_scripts/before_install_docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
can you use `set -e` to fail on the first non-zero exit code in all the bash scripts ?
I think that more closely mirrors what travis is doing, and should abort the test.

Reply:
Yeah I will do. We found the BIOBLEND_GALAXY_URL bug almost in the same time ;-).

I am happy to really test the ansible install now ;-)
Next step I would like to test Ubuntu 16.04 too.  This can only be done in docker container if using Travis, right ? Maybe other services could provide Ubuntu 16.04 VM ? What about Drone ?